openbb_sdk/extensions/charting/openbb_charting/core/plotly.html

```diff
@@ -4083,7 +4083,7 @@
AAAAUAOAAFADgABAAAARgABAAAAAQAIAAE/9P9QAAEAAAABAAgAAT/mAEYABgAAAAEACAABAK4AAwAMACIATAABAAQAAQZzAAEAAgTFBMUAAQAAAEgAAgAGABgAAwTUBLQGcwABAAAAAQAAAEgAAgS0BnMAAQABBMUAAQAAAEkABAAKABoAKgA8AAEGLQABAAMEbATFBMUAAAAAAAEABARsBMUExQTFAAAAAgRyBNgAAQADBGwExQTFAAAAAAABAAMEbATFBMUAAQAAAEgAAQAAAAEACAACAAwAAwS0BXAGcwABAAMEbATFBi0ABgAAAAEACAABDTAAAgAKAB4AAQAEAAIGdAZ0AAEAAAABAAAASwAEAAoFegAYACgAAQYyAAEAAgYyBiwAAAABBnQAAQABBiwAAQAAAEsAAAABAAIGMgYsAAEAAABMAAEAAAABAAgAAgzSAAIFiAZ0AAYAAAABAAgAATE6AAEACAAHABAAHgAsADoASABYAGgAAQR5AAEAAgR5BHkAAAAAAAEAAwR5BHkEeQAAAAEEigABAAIEeQR5AAAAAAABAAMEeQR5BIoAAAACBLYEtgABAAAAAQAAAE4AAQS2AAEAAQR5AAEAAABPAAAAAQACBHkEeQABAAAAUAABAAAAAQAIAAEwsgACAAYAAAABAAgAAjHCEAgxHBAqAAI6IgAQAAgAEgAgAC4APgBMAFoAagB6AAEABQABAAIAAgACAAAAAAABAAMAAgACAAIAAAACAAIAAwABAAIAAgACAAAAAAABAAMAAgACAAEAAAABAAEAAQACAAIAAgAAAAIABAAEAAEAAAABAAAAUgABAAQAAQABAAIAAQAAAFMAAAABAAIAAgACAAEAAABUAAEAAAABAAgAATEgAAIABgAAAAEACAABEDwAAgAKAGIABQAMABoAKAA4AEgAAQRkAAEAAgRkBioAAAAAAAEAAwRkBioGKgAAAAIEcgTYAAEAAgRkBioAAAABBLIAAQABBioAAQAAAFYAAAABAAIEZAYqAAEAAABXAAEABAACBLIEsgABAAAAAQAAAFYAAQAAAAEACAACD74AAgSyBGcABgAAAA4AIgA0AEYAWgBwAIQAlgCoALwAzgDgAPQBCAEcAAMAATyyAAE8sgACPLI8sgAAAAMAAAABPKAAAzygPKA8oAAAAAMAAh1yEDwAATyOAAI8jjyOAAAAAwADPIAdXhAoAAE8egACPHo8egAAAAMAAh1IPGoAATxkAAI8ZDxkAAAAAwABECYAATxQAAI8UDxQAAAAAwAAAAE8PgADPD48PhAqAAAAAwACEGI8LAABPCwAAjwsPCwAAAADAAEnUAABPBgAAjwYPBgAAAADAAAAATwGAAM8BjwGJz4AAAADAAAAATv0AAQ79Dv0ECo79AAAAAMAAjvIO8gAATvgAAAAAQAAAFkAAwABO7QAATvMAAE7zAABAAAAWgADAAAAATu4AAI7uDu4AAEAAABbAAEAAAABAAgAATuc/0IABgAAAAEACAABEJQAAgAKAHQABgAOABwAKgA6AEwAWgABBGwAAQACBioGKgAAAAAAAQADBioGKgYqAAAAAgRyBNgAAQACBioGKgAAAAMGLQRyBNgAAQACBioGKgAAAAEGKgABAAIGKgYqAAAAAAABAAIGKgYqAAEAAABdAAIABgAWAAIGcQS0AAEAAAABAAAAXQABBLQAAQABBioAAQAAAF4AAQAAAAEACAACD/IAAgS0BHAABgAAAAEACAABAJwAAgAKAB4AAQAEAAIEtAS0AAEAAAABAAAAYAAGAA4AHAAqADoATABcAAEEbAABAAIEbARdAAAAAAABAAMEbARdBF0AAAACBHIE2AABAAIEbARdAAAAAwYtBHIE2AABAAIEbARdAAAAAQS0AAEAAQRdAAEAAABgAAAAAQACBGwEXQABAAAAYQABAAAAAQAIAAIACgACBG4EtAABAAIEXQRsAAYAAAABAAgAARGsAAIACgA+AAMACAAWACQAAQYtAAEAAgYyBjIAAAAAAAEAAwYyBjIGMgAAAAAAAQACBjIGMgABAAAAYwACAAYAFgACBnQGcwABAAAAAQAAAGMAAQZzAAEAAQYyAAEAAABkAAEAAAABAAgAAhFAAAIGcwWBAAYAAAABAAgAAQB8AAMADAAgAFQAAQAEAAIGdAZzAAEAAAABAAAAZgADAAgAFgAkAAEGLQABAAIGMgYsAAAAAAABAAMGMgYsBiwAAAAAAAEAAgYyBiwAAQAAAGYAAQAEAAEGcwABAAEGLAABAAAAZgABAAAAAQAIAAIADAADBYAGcwZ0AAEAAwYsBi0GMgAGAAAAAQAIAAEAXAADAAwAIAA0AAEABAABBnMAAQABBiwAAQAAAGgAAQAEAAIEtgZzAAEAAAABAAAAaAABAAQAAAABAAIEeQYsAAEAAABoAAEAAAABAAgAAgAMAAMEtgVyBnMAAQADBHkGLAYtAAYAAAABAAgAAQBcAAMADAAgADQAAQAEAAEGcwABAAEGLAABAAAAagABAAQAAgX0BnMAAQAAAAEAAABqAAEABAAAAAEAAgXsBiwAAQAAAGoAAQAAAAEACAACAAwAAwX0BXgGcwABAAMF7AYsBi0ABgAAAAEACAABAFwAAwAMACAANAABAAQAAQZzAAEAAQYsAAEAAABsAAEABAACBnAGcwABAAAAAQAAAGwAAQAEAAAAAQACBiYGLAABAAAAbAABAAAAAQAIAAIADAADBnAFegZzAAEAAwYmBiwGLQAGAAAAAQAIAAEAfAADAAwAIAA0AAEABAABBnMAAQABBiwAAQAAAG4AAQAEAAIEuAZzAAEAAAABAAAAbgADAAgAFgAkAAEGLQABAAIEigYsAAAAAAABAAMEigYsBiwAAAAAAAEAAgSKBiwAAQAAAG4AAQAAAAEACAACAAwAAwS4BYMGcwABAAMEigYsBi0ABgAAAAEACAABAHwAAwAMAEAAVAADA
```

Comment:
Why is this changed?

Reply:
It had eod somewhere so it got changed. Reverting it.